### PR TITLE
DevEx - interactors calling interactors

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/useCaseHelper/addCoverToPdf.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * a helper function which creates a coversheet, prepends it to a pdf, and returns the new pdf
+ *
+ * @param {object} options the providers object
+ * @param {object} options.applicationContext the application context
+ * @param {string} options.caseEntity the case entity associated with the document we are creating the cover for
+ * @param {object} options.documentEntity the document entity we are creating the cover for
+ * @param {object} options.pdfData the original document pdf data
+ * @returns {object} the new pdf with a coversheet attached
+ */
+exports.addCoverToPdf = async ({
```
Comment (kkoskelin) [inline shared/src/business/useCaseHelper/addCoverToPdf.js:19]: test coverage?

File: shared/src/business/useCaseHelper/generateCoverSheetData.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  const dateServedFormatted =
+    (documentEntity.servedAt &&
+      'Served ' +
+        applicationContext
+          .getUtilities()
+          .formatDateString(documentEntity.servedAt, 'MMDDYYYY')) ||
+    '';
+
+  let dateReceivedFormatted;
```
Comment (kkoskelin) [inline shared/src/business/useCaseHelper/generateCoverSheetData.js:28]: nit: I might have written a ternary that chose the date formatting preference, but it's cool.

File: shared/src/business/useCaseHelper/generateDocketRecordPdf.js [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {string} providers.caseId the case id for the docket record to be generated
+ * @returns {Uint8Array} docket record pdf
+ */
+exports.generateDocketRecordPdf = async ({
+  applicationContext,
+  caseId,
+  docketRecordSort,
+  includePartyDetail = false,
```
Comment (kkoskelin) [inline shared/src/business/useCaseHelper/generateDocketRecordPdf.js:16]: These two missing from jsdoc

File: web-api/src/genericHandler.js [hunk 4]
```diff
@@ -85,6 +85,9 @@ exports.genericHandler = (event, cb, options = {}) => {
 
       return returnResults;
     } catch (e) {
+      console.log('e', e);
+      console.log('e', e);
+      console.log('e', e);
```
Comment (kkoskelin) [inline web-api/src/genericHandler.js:90]: Perhaps you wanted to remove these.

File: web-client/integration-tests-public/unauthedUserSearchesForOrder.test.js [hunk 5]
```diff
@@ -44,6 +44,8 @@ describe('Petitioner creates case', () => {
       partyType: ContactFactory.PARTY_TYPES.petitionerSpouse,
     });
     test.docketNumber = caseDetail.docketNumber;
+
+    console.log('test.docketNumber', test.docketNumber);
```
Comment (kkoskelin) [inline web-client/integration-tests-public/unauthedUserSearchesForOrder.test.js:48]: remove logging

File: web-client/integration-tests/journey/docketClerkViewsQCInProgress.js [hunk 6]
```diff
@@ -15,7 +15,7 @@ export const docketClerkViewsQCInProgress = (test, shouldExist) => {
     expect(workQueueToDisplay.box).toEqual('inProgress');
 
     const inProgressQueue = test.getState('workQueue');
-    console.log(''); // adding this causes the tests to pass....
+    console.log(''); // TODO: potentially remove?  adding this causes the tests to pass....
```
Comment (kkoskelin) [inline web-client/integration-tests/journey/docketClerkViewsQCInProgress.js:18]: yikes.

**Review by kkoskelin — APPROVED** [review]
Perhaps a couple logging statements to remove, but ... nice.

**Comment by codyseibert** [comment]
too many conflicts, it will be easier to just redo this work

**Pull request closed without merging**